api/base/settings/defaults.py

```diff
@@ -104,6 +104,16 @@
         'api.base.authentication.drf.OSFSessionAuthentication',
         'api.base.authentication.drf.OSFCASAuthentication'
     ),
+    'DEFAULT_THROTTLE_CLASSES': (
+        'rest_framework.throttling.UserRateThrottle',
+        'api.base.throttling.NonCookieAuthThrottle',
+        'api.base.throttling.AddContributorThrottle',
+    ),
+    'DEFAULT_THROTTLE_RATES': {
+        'user': '10000/day',
+        'non-cookie-auth': '100/hour',
+        'add-contributor': '10/hour',
```

Comment:
Based on what I've seen with this, I going to go on a limb and say I think you believe that the rates are tied to the default classes, and I don't *think* that's true. I think the default throttle rates are just a dictionary used by the throttle classes so that, if there's a scope, it'll see if there's a default rate in this dictionary that's keyed to that scope. I think you could have two default throttle classes and 200 default throttle rates and there proooobably wouldn't be a problem. 

Unless you've tried that and I'm completely wrong. But that was how I was imagining it.